frontend/app/src/pages/balances/blockchain/index.vue

```diff
@@ -197,6 +204,26 @@ const { createAccount, editAccount } = useAccountDialog();
       />
     </card>
 
+    <div class="mt-8">
+      <v-tooltip right max-width="300">
+        <template #activator="{ on }">
+          <v-btn
+            color="primary"
+            depressed
+            :loading="isEvmAccountsDetecting"
+            :disabled="isEvmAccountsDetecting"
+            v-on="on"
+            @click="detectEvmAccounts"
+          >
+            {{ tc('blockchain_balances.evm_detection.title') }}
+          </v-btn>
+        </template>
+        <span>
+          {{ tc('blockchain_balances.evm_detection.tooltip') }}
+        </span>
+      </v-tooltip>
+    </div>
```

Comment:
This can be a self-contained component right? can you extract it into a new component?